python/core/auto_generated/layertree/qgslayertreeregistrybridge.sip.in

```diff
@@ -41,10 +49,20 @@ Create the instance that synchronizes given project with a layer tree root
     void setNewLayersVisible( bool enabled );
     bool newLayersVisible() const;
 
-    void setLayerInsertionPoint( QgsLayerTreeGroup *parentGroup, int index );
+ void setLayerInsertionPoint( QgsLayerTreeGroup *parentGroup, int index ) /Deprecated/;
 %Docstring
 Set where the new layers should be inserted - can be used to follow current selection.
 By default it is root group with zero index.
+
+.. deprecated:: since QGIS 3.10
```

Comment:
add a note to use setLayerInsertionPoint( const InsertionPoint &insertionPoint ) instead.